package.json

```
@@ -107,6 +113,6 @@
     }
   },
   "engines": {
-    "node": ">= 18"
+    "node": "^18 || >= 20"
```

```suggestion
    "node": "^18.12 || ^20.9"
```

Pour cibler spécifiquement les versions LTS de Node.